tests/test_calendar_grid.py

```diff
@@ -0,0 +1,94 @@
+import datetime
+
+import app.routers.calendar_grid as cg
+
+DATE = datetime.date(1988, 5, 3)
+DAY = cg.Day(datetime.date(1988, 5, 3))
+WEEKEND = cg.DayWeekend(datetime.date(2021, 1, 23))
+N_DAYS = 3
+N_DAYS_BEFORE = datetime.date(1988, 4, 30)
+NEXT_N_DAYS = [
+    cg.Day(datetime.date(1988, 5, 4)),
+    cg.Day(datetime.date(1988, 5, 5)),
+    cg.Day(datetime.date(1988, 5, 6))
+]
+DAY_TYPES = [cg.Day, cg.DayWeekend, cg.Today, cg.FirstDayMonth]
+
+
+class TestCalendarGrid:
+    @staticmethod
+    def test_get_calendar(client):
+        response = client.get("/calendar")
+        assert response.status_code == 200
```

Comment:
Prefer: `.ok`

Reply:
I changed it to use the builtin module "http" instead.